tests/acceptance/features/bootstrap/UserLdapGeneralContext.php

```diff
@@ -393,7 +393,7 @@ public function connectToLdap($suiteParameters) {
 		$this->ldap->bind();
 		SetupHelper::init(
 			"admin", (string)$suiteParameters['adminPassword'],
```

Comment:
This whole call should be like:
```
		SetupHelper::init(
			$this->featureContext->getAdminUsername(),
			$this->featureContext->getAdminPassword(),
			$this->featureContext->getBaseUrl(),
			$this->featureContext->getOcPath()
		);
```
and you can also remove ``adminUsername`` and ``adminPassword`` from the suite parameters in ``behat.yml``